services/crypto_update.proto

```diff
@@ -129,4 +129,27 @@ message CryptoUpdateTransactionBody {
      * including implicit and explicit associations.
      */
     google.protobuf.Int32Value max_automatic_token_associations = 15;
+
+    /**
+     * ID of the account or node to which this account is staking.
+     */
+    oneof staked_id {
+
+        /**
+         * ID of the new account to which this account is staking. If set to the sentinel <tt>0.0.0</tt> AccountID,
+         * this field removes this account's staked account ID.
+         */
+        AccountID staked_account_id = 16;
+
+        /**
+         * ID of the new node this account is staked to. If set to the sentinel <tt>-1</tt>, this field
+         * removes this account's staked node ID.
+         */
+        google.protobuf.Int64Value staked_node_id = 17;
```

Comment:
I did some local testing. Having an `Int64Value` does not provide any value here. The wrapper type's main purpose is to allow set versus unset semantics for primitive types. But here, the `oneof` already provides that capability since by default the `oneof` has a state of unset. The wrapper does not help with being able to clear a value since it will throw a NPE if you try to set staked_account_id or stake_node_id to null in the builder. Recommend we change back to primitive for updates:

```suggestion
        int64 staked_node_id = 17;
```